int64 tension
int64 motor_number
---
bool result
